bitnami/redis/Chart.yaml

```
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: redis
-version: 11.1.4
+version: 11.1.5
```

As it's a feature addition instead of a bug fix, I would bump the chart version in a minor instead of a patch
```suggestion
version: 11.2.0
```

---

Will do, thanks!